bot/code_review_bot/tasks/infer.py

```diff
@@ -21,10 +21,6 @@
 - **Infer check**: {check}
 - **Publishable **: {publishable}
 - **Is new**: {is_new}
-
-```
-{body}
```

Comment:
No body?

Reply:
It was always `None`, and only used in the admin mail